phy/cluster/algorithms.py

```diff
@@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
     def _create_filter(self):
         rate = self._kwargs['sample_rate']
         low = self._kwargs['filter_low']
```

Comment:
Disagree - it takes a long time to even just load the LFP from disk (last
night, e.g., the one I was working with was 5000000x129 (int16) = 1.5GB).
You definitely don't want to load the 16x larger raw data and compute the
LFP from it when going to analyze that. If you're already loading the raw
data to detect spikes, this seems like a fine time to filter and save the
LFP.

On Tue, Jun 30, 2015 at 6:38 PM, nippoo <notifications@github.com> wrote:

> In phy/cluster/algorithms.py
> <https://github.com/kwikteam/phy/pull/448#discussion_r33600813>:
>
> > @@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
> >      def _create_filter(self):
> >          rate = self._kwargs['sample_rate']
> >          low = self._kwargs['filter_low']
>
> I agree - both should be in Hz.
>
> For the .low.kwd - I don't think we need to extract LFP at the time of
> spike detection. We should be able to do this later, at the time of
> analysis.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kwikteam/phy/pull/448/files#r33600813>.
>